docs/pyrona.md

```diff
@@ -0,0 +1,147 @@
+FrankenScript is a small language that is designed to be easy to modify and extend.
+It is based on the ideas of ownership and concurrency that we are exploring in Project Verona, but where all the checks are dynamic.
+
+This prototype has given us confidence in the conceptual ideas behind our ownership model, and has allowed us to explore the design space of ownership in a dynamic language.
+It has also helped us explain our ideas and the different design decisions to people outside the Project Verona team.
+
+
+FrankenScript generates a series of mermaid diagrams that show the region structure of the program.  An example of this is shown below:
+```mermaid
+%%{init: {'theme': 'neutral', 'themeVariables': { 'fontSize': '16px' }}}%%
+graph TD
+  id0(None):::immutable
+id1[0x60800000cfa0&#13;<br/>rc=1]
+id1--> |#95;#95;proto#95;#95;| id2[#91;Frame#93;&#13;<br/>rc=2 #40;global#41;]
+  linkStyle 0 stroke:#777777,stroke-width:2px
+id1-.-> |z| id3[0x607000009a20&#13;<br/>rc=2]
+  linkStyle 1 stroke:orange,stroke-width:2px
+id3--> |a| id4[0x607000009cc0&#13;<br/>rc=1]
+  linkStyle 2 stroke:#777777,stroke-width:2px
+id1-.-> |r| id5[\0x607000009f60&#13;<br/>lrc=2&#13;<br/>sbrc=0&#13;<br/>rc=1/]
+  linkStyle 3 stroke:orange,stroke-width:2px
+id5--> |#95;#95;proto#95;#95;| id6(#91;RegionObject#93;&#13;<br/>rc=2 #40;global#41;):::immutable
+  linkStyle 4 stroke:#76c5cc,stroke-width:2px
+id5--> |f| id3
+  linkStyle 5 stroke:#777777,stroke-width:2px
+id3
+id4
+id5
+id1
+subgraph LocalReg["Local region"]
+  id1
+  id2
+end
+style LocalReg fill:#eefcdd
+subgraph reg0x60800000d420[" "]
+  id3
+  id4
+  id5
+end
+style reg0x60800000d420 fill:#fcfbdd
+classDef unreachable stroke-width:2px,stroke:red
+classDef highlight stroke-width:4px,stroke:yellow
+classDef error stroke-width:4px,stroke:red
+classDef tainted fill:#8e84cc
+classDef immutable fill:#94f7ff
+```
+
+
+### Engaging with the Python community
+
+Over the last two years, we have been engaging with the Faster CPython team at Microsoft as a sounding board for our ideas.
+In May 2025, we will be taking our first steps into the boarder Python community at the [Python Language Summit](https://us.pycon.org/2025/events/language-summit/).
+We will be presenting our ideas and seeking feedback from the core developers of the language.
+
+
+### Steps to a new ownership model
+
+Building an ownership model for Python is a complex task, and we are taking a step-by-step approach to ensure that we get it right.
+
+The first step is actually to build a concept of deep immutability into Python.  This can be split into three parts:
+
+* Deep Immutability: We are starting with a deep immutability model, we have been drafting a [PEP](https://github.com/TobiasWrigstad/peps/pull/8) to describe this model.
+  The current prototype is in a PR to a fork of CPython: https://github.com/mjp41/cpython/pull/51
```

Comment:
Should we mention here that it's on 3.12 and that the pep has a section about rebasing? Just to prevent any critique in that fashion?

Or maybe better, update the description of that PR to make clear that we're aware of this